run_kraken.py

```diff
@@ -374,10 +380,10 @@ def main(cfg):
         # if platform is openshift will be collected
         # Cloud platform and network plugins metadata
         # through OCP specific APIs
-        if config["kraken"]["distribution"] == "openshift":
-            telemetry_ocp.collect_cluster_metadata(chaos_telemetry)
-        else:
-            telemetry_k8s.collect_cluster_metadata(chaos_telemetry)
+        # if config["kraken"]["distribution"] == "openshift":
```

Comment:
Did we disable it for testing purposes? Thoughts?

Reply:
yeah, an oversight, restoring.